_features/discoverable.html

```diff
@@ -2,34 +2,22 @@
 title: Discoverable
 slug: discoverable
 description: Are identifiable as applications and search engines can find them.
+usecases:
+- <p>Progressive Web Applications offer unique capabilities; as a developer, I want
+  my web applications to be promoted in search engines when the user is using
+  modern platforms.</p>
+- <p>As a search engine owner I want to highlight those applications with best UX
```

Comment:
Nit: here and below, append a comma to the "as a … clause, i.e. `As a search engine owner,`.